test/BitcoinHelpersTest.js

```diff
@@ -1,51 +1,45 @@
-/*
-const BitcoinAddress = require("../src/Address")
-const Network = BitcoinAddress.Network
-
-const chai = require("chai")
-const assert = chai.assert
+import BitcoinHelpers from "../src/BitcoinHelpers.js"
+import { assert as _assert } from "chai"
+const assert = _assert
```

Comment:
Why not just `import { assert } from "chai"`?

Reply:
Yes, it works. Fixed it in https://github.com/keep-network/tbtc.js/pull/133/commits/efe26da71067f1bc7120a7086b6b5f55b2e62ea8.

Reply:
Whoops, but looks like this file didn't change in that commit :grimacing:

Reply:
Yes, I omitted one import. Fixed in https://github.com/keep-network/tbtc.js/pull/133/commits/e94380a3223b8db8feca53ec0c4f9e1d50d69773.